core-tests/src/test/java/org/mule/runtime/core/internal/processor/interceptor/ReactiveInterceptorAdapterTestCase.java

```diff
@@ -816,6 +830,48 @@ public void interceptedThrowsException() throws Exception {
     }
   }
 
+  @Test
+  @Issue("MULE-19525")
```

Comment:
this issue is just adding a test, this should refer to the bug this tests was added for